### PR TITLE
debug: useContext bug in 'Page'

### PR DESCRIPTION
**Pull request by SimpleCookie**
I don't know what is causing this, the bug is really weird indeed. Though, I'm not good at react context.
But this is a workaround, and perhaps it can help you figure out what the problem is.

You can decline this PR if you want. I just made it so that you can read my comments :)
I'll add some comments.

### CODE REVIEW

File: examples/next-starter/src/app/components/PageContainer.tsx [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+import { AppContext } from '@onr/core';
 import { Page } from '@onr/core-antd';
 import { AccountSelector } from '@onr/plugin-account';
 import { AppProps } from 'next/app';
+import { useContext } from 'react';
 import { theme } from './GlobalStyles';
 
 const Container: React.FC = (props: AppProps) => {
   const { Component, pageProps } = props;
+  const context = useContext(AppContext)
```
Comment (SimpleCookie) [inline examples/next-starter/src/app/components/PageContainer.tsx:11]: ` const context = useContext(AppContext)` works fine here, that's no problem.

File: packages/core-antd/src/components/layout/Page.tsx [hunk 2]
```diff
@@ -13,29 +13,35 @@ interface Props {
...
 }
 
 const { Content } = Layout;
 
 /* eslint-disable complexity */
 export const Page = (props: Props) => {
-  const { HeaderMainSection, theme, logout, children } = props;
+  const { HeaderMainSection, theme, logout, children, context } = props;
   const router = useRouter();
-  const appConfig = useContext(AppContext).getAppConfig();
```
Comment (SimpleCookie) [inline packages/core-antd/src/components/layout/Page.tsx:24]: ` const context = useContext(AppContext)`  does **not** work here.
And when you ctrl+click on AppContext to enter the implementation, you go to `packages\core-antd\node_modules\@onr\core\src\components\App.tsx`

Maybe the path is the reason, it's importing the wrong class. I don't know why it's doing this though.

Comment (SimpleCookie) [reply packages/core-antd/src/components/layout/Page.tsx:24]: I assume the problem is related to this Lerna thingy 😄

File: packages/core-antd/src/components/layout/Page.tsx [hunk 3]
```diff
@@ -13,29 +13,35 @@ interface Props {
...
 
   useEffect(() => {
-    setTimeout(() => {
+    const timeout = setTimeout(() => {
       setLoading(false);
     }, 1000);
+    () => clearTimeout(timeout)
   }, [loading]);
 
+  if (!context) return null
```
Comment (SimpleCookie) [inline packages/core-antd/src/components/layout/Page.tsx:39]: I added this just for testing purposes

**Comment by koshuang** [comment]
@SimpleCookie thank you for your time :) I will check tonight when I start live streaming.

File: packages/core-antd/src/components/layout/Page.tsx (hunk 2 above)
Comment (koshuang) [reply packages/core-antd/src/components/layout/Page.tsx:24]: Good point! I think it's very possible that it's using wrong class or instance with same name. That gives me a direction. Thanks!

**Pull request closed without merging**